moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -89,29 +89,30 @@ servo:
 ################################ GENERAL CONFIG #############################
   enable_parameter_update: {
     type: bool,
-    default_value: false,
+    default_value: true,
     description: "If true, continuous check for parameter update is enabled \
                   Currently only the following parameters are updated:
                   1. override_velocity_scaling_factor
                   2. robot_link_command_frame"
   }
 
-  scale:
-    linear: {
-      type: double,
-      default_value: 0.4,
-      description: "Max linear velocity. Unit is [m/s]. Only used for Cartesian commands."
-    }
-    rotational: {
-      type: double,
-      default_value: 0.8,
-      description: "Max angular velocity. Unit is [rad/s]. Only used for Cartesian commands."
-    }
-    joint: {
-      type: double,
-      default_value: 0.5,
-      description: "Max joint angular/linear velocity. Only used for joint commands on joint_command_in_topic."
-    }
+  linear_scale: {
+    type: double,
+    default_value: 0.4,
+    description: "Max linear velocity. Unit is [m/s]. Only used for Cartesian commands."
+  }
+
+  rotational_scale: {
+    type: double,
+    default_value: 0.8,
+    description: "Max angular velocity. Unit is [rad/s]. Only used for Cartesian commands"
+  }
+
+  joint_scale: {
+    type: double,
+    default_value: 0.5,
+    description: "Max joint angular/linear velocity. Only used for joint commands on joint_command_in_topic"
+  }
```

Comment:
If you merge/rebase on main, these params should retain the original structure

Reply:
These are unused now since support for unitless command is not added yet, so I removed them in the last update.
I am not yet sure on how those should be handled. It does not make sense for servo to directly receive a "unitless" command. 
This should probably be handled by a utility function before the command is passed to servo.